beetsplug/web/__init__.py

```diff
@@ -310,6 +314,9 @@ def func(lib, opts, args):
                 self.config['port'] = int(args.pop(0))
 
             app.config['lib'] = lib
+            # Normalizes json output
+            app.config['JSONIFY_PRETTYPRINT_REGULAR'] = False
```

Comment:
After experiencing with the json output of jsonify and json.dumps I noticed that the output of json.dumps is always compact while jsonify outputs it prettified.

That's the first problem, the second is that a prettified json string takes mus more space. Even the compat string of my library is already > 30 MB.